doc/api/index.rst

```diff
@@ -147,3 +129,54 @@ Matplotlib. The following toolkits are included:
    toolkits/axes_grid1.rst
    toolkits/axisartist.rst
    toolkits/axes_grid.rst
+
+
+.. _usage_patterns:
+
+Usage patterns
+--------------
+
+Below we describe several common approaches to plotting with Matplotlib.
+
+The pyplot API
+^^^^^^^^^^^^^^
+
+`matplotlib.pyplot` is a collection of command style functions that make
```

Comment:
```suggestion
`matplotlib.pyplot` is a collection of functions that make
```
Maybe it's just me, but I have no idea what "command style functions" are. I feel it's still (or even better) understandable with only "functions".

Reply:
Sure, I just cut and pasted this...